data/templates/high-availability/keepalived.conf.j2

```diff
@@ -128,7 +128,7 @@ vrrp_instance {{ name }} {
 {%             endfor %}
     }
 {%         endif %}
-{%         if group_config.health_check is vyos_defined %}
+{%         if group_config.health_check is vyos_defined and group_config.health_check.script is not vyos_defined %}
```

Comment:
`{%         if group_config.health_check.script is not vyos_defined %}` would be sufficient in this case